src/Components/FeelingRoom/FeelingRoom.js

```diff
@@ -78,27 +92,44 @@ class FeelingRoom extends Component {
   startFeeling = () => {
     this.setState({isFeeling: true})
   }
+  playSound(e) {
+    let audio = e.target.parentNode.parentNode.firstChild
+    audio.play()
+  }
+  pauseSound(e) {
+    let audio = e.target.parentNode.parentNode.firstChild
+    audio.pause()
+  }
+
   render(){
     let instructions = (
-      <section className='Directions'>
+      <section className='directions'>
         <h1> Welcome to the feeling room </h1>
         <p>
           First pick a kind of sound you want to listen to.
           Then feel free to pause and play sound as you see fit.
           Enjoy the sights and sounds of your feeling space.
         </p>
-        <button onClick={this.startFeeling}> Get Started </button>
+        <button onClick={this.startFeeling} className='getStarted'> Get Started </button>
       </section>
     )
     return (
       <section className='FeelingRoom'>
         {!this.state.isFeeling && instructions}
         <section className="soundControlPanel">
-          <button>Nature Sound Svg</button>
-          <button>Ambient Sound Svg</button>
-          <button>Music Sound Svg</button>
-          <button> Play </button>
-          <button> Pause </button>
+          <div className='chooseSound'>
```

Comment:
Absolutely love the styling on the menu modal! It's exactly what we talked about, and it looks really good. I think it should mesh well with the styling of the rooms, no matter the mood.